### DISCUSSION TITLE
How do I disable the version updater close app prompt?

### DISCUSSION THREAD

**Discussion by givihuda** [dbeaver/dbeaver#20050] [Q&A]
Every time I click to update and DBeaver needs to close to do it, I just want it to do it.

I don't want this prompt every time:

![image](https://github.com/dbeaver/dbeaver/assets/74785256/a63b4765-e0bd-4ab1-98f1-803fa2f741d4)

There should be a checkbox to "Always close DBeaver to install the update when it needs to" or something like that, that remembers when I click "Yes".

**Comment by LonwoLonwo** [comment]
Hello. We can probably add it to the Confirmations questions list. You can create a feature request ticket for this.
